docs/api/apiv3/components/schemas/attachments_model.yml

```diff
@@ -0,0 +1,15 @@
+# Schema: Attachments
+---
+type: object
+properties:
```

Comment:
Here, the same problem hits like in #10179, I'd like to get rid of those redundant schema declarations for collections step by step.

Can you cast another glance, how those classes are usable from your generated client?

Reply:
Not quite sure what you mean - the `allOf` approach to integrate the `collection_model.yml`, which we're discussing in [PR 10179](https://github.com/opf/openproject/pull/10179#discussion_r830523691), or something else?

Reply:
Yes, sorry for being not explicit.

The current state looks fine, except this last small thing.

Reply:
yes, those different attachment models were basically generated automatically, when we created the initial version of the spec from grape.

But there are no differences in the attachment models, we can unitfy them. Thanks for your effort.